client/packages/programs/src/JsonForms/common/components/CategorizationTabLayout.tsx

```diff
@@ -95,6 +99,75 @@ const CategoryLayoutRendererComponent = ({
 };
 const CategoryLayoutRenderer = React.memo(CategoryLayoutRendererComponent);
 
+// Try to extract a more precise error path
+const propertyPathFromError = (error: {
+  instancePath: string;
+  keyword: string;
+  params: Record<string, any>;
+}): string => {
+  // see https://ajv.js.org/api.html#error-objects
+  switch (error.keyword) {
+    case 'dependencies':
+      return `${error.instancePath}/${error.params['missingProperty']}`;
+    case 'required':
+      return `${error.instancePath}/${error.params['missingProperty']}`;
+    case 'propertyNames':
+      return `${error.instancePath}/${error.params['propertyName']}`;
+    default:
+      return error.instancePath;
+  }
+};
+
+// Recursively goes through a layout and collects all element paths that have errors
+const containsErrors = (layout: Layout, errorPaths: string[]): string[] => {
+  const results = [];
+  for (const element of layout.elements) {
+    if (isScoped(element)) {
+      const scopePath = toDataPathSegments(element.scope).reduce(
+        (prev, current) => {
+          return `${prev}/${current}`;
+        },
+        ''
+      );
+      for (const errorPath of errorPaths) {
+        if (scopePath.startsWith(errorPath)) {
+          results.push(errorPath);
+        }
+      }
+    } else if (isLayout(element)) {
+      results.push(...containsErrors(element, errorPaths));
+    }
+  }
+  return results;
+};
+
+const ErrorStringComponent: FC<{
+  category: Category;
+  errorPaths: string[];
+}> = ({ category, errorPaths }) => {
+  const t = useTranslation('common');
+
+  if (errorPaths.length === 0) {
+    return null;
+  }
+  const foundPaths = containsErrors(category, errorPaths);
+  if (foundPaths.length === 0) {
+    return null;
+  }
+  return (
+    <Typography
+      sx={{
+        color: theme => theme.palette.error.main,
+        backgroundColor: theme => theme.palette.background.login,
+        borderRadius: '16px',
+        padding: '0px 8px',
```

Comment:
In order to remove hard coded pixel values 🤷 
```suggestion
        borderRadius: 4,
        paddingX: 2,
```